metadata-ingestion/src/datahub/utilities/source_helpers.py

```diff
@@ -148,3 +155,44 @@ def auto_materialize_referenced_tags(
             entityUrn=urn,
             aspect=TagKeyClass(name=tag_urn.get_entity_id()[0]),
         ).as_workunit()
+
+
+def auto_browse_path_v2(
+    stream: Iterable[MetadataWorkUnit],
+) -> Iterable[MetadataWorkUnit]:
+    """Generate BrowsePathsV2 from Container and BrowsePaths aspects."""
```

Comment:
we don't seem to be inspecting the `BrowsePaths` aspect here?

Reply:
Going to do this in another PR, but updated the comment